deployments/quick-start/kubernetes-vault-rbac.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-vault
+  namespace: default
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: kube-vault
+rules:
+- apiGroups: [""]
+  resources:
+  - pods
+  verbs: ["get", "list"]
```

Comment:
Could you please double check: https://github.com/Boostport/kubernetes-vault/blob/master/controller/client/kube.go is this file correct? I see list pods and watch for pods, I also see get endpoints further down.

To answer your second question, I have the impression that customers use this to watch pods in more than one namespace, is this not the case? Is there a reason why it should only apply in one namespace? If the general rule is to apply to only one namespace and only one namespace (which I could totally understand for security sake) then maybe instead of a cluster role we should be applying a Role instead. Alternatively if customers use both cases then maybe we could apply options for them to use in either scenario.